README.md

```diff
@@ -298,6 +298,34 @@ The policy giving access to the role running Waggle Dance will need at least the
     "glue:GetUserDefinedFunctions"
     ]
 
+##### Federate to AWS Glue Catalog for writes
+
+Writes to Glue are supported as best effort via the same glue library as is used for reads.
+This maps the most common thrift methods to representative Glue calls. Not everything is support as Glue doesn't support the full thrift stacks as it's not HMS. In some cases it will be better to use direct glue access through EMR or connecting to HMS directly.
+
+Basic stuff is tested and works: creating tables, adding partitions, dropping tables, alter tables.
+
+Deployment changes to support this functionality:
+* Expand your Glue policy to allow for create/update operations, search AWS documentation for most up to date list.
+* Add an S3 policy that allows for object reading and creating. Similar permissions as HMS would have. This is needed because upon table creating the table location path will be created in S3. Note that this would normally happen in HMS and will now happen in Waggle Dance.
```

Comment:
```suggestion
* Add an S3 policy that allows for object reading and creating. Similar permissions as HMS would have. This is needed because upon table creation, the table location path will be created in S3. Note that this would normally happen in HMS and will now happen in Waggle Dance.
```